xml/CRSChannelCode.xml

```diff
@@ -9,2494 +9,3284 @@
-            <code>47035</code>
+        <codelist-item status="active" activation-date="2011-01-01">
+            <code>21006</code>
             <name>
-                <narrative>Environmental Development Action in the Third World</narrative>
-                <narrative xml:lang="fr">Environnement et développement du Tiers-monde</narrative>
+                <narrative>Development Gateway Foundation</narrative>
+                <narrative xml:lang="fr">Development Gateway Foundation</narrative>
             </name>
+            <category>22000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
+        <codelist-item status="withdrawn" activation-date="2010-01-01" withdrawal-date="2010-12-31">
             <code>21007</code>
             <name>
                 <narrative>Environmental Liaison Centre International</narrative>
                 <narrative xml:lang="fr">Centre international de liaison pour l’environnement</narrative>
             </name>
+            <category>21000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
-            <code>21503</code>
+        <codelist-item status="active" activation-date="2011-01-01">
+            <code>21008</code>
             <name>
-                <narrative>Family Health International 360</narrative>
+                <narrative>Eurostep</narrative>
+                <narrative xml:lang="fr">Eurostep</narrative>
+            </name>
+            <category>22000</category>
+        </codelist-item>
+        <codelist-item status="active" activation-date="2011-01-01">
+            <code>21009</code>
+            <name>
+                <narrative>Forum for Agricultural Research in Africa</narrative>
+                <narrative xml:lang="fr">Forum pour la recherche agricole en Afrique</narrative>
+            </name>
+            <category>51000</category>
+        </codelist-item>
+        <codelist-item status="active" activation-date="2011-01-01">
+            <code>21010</code>
+            <name>
+                <narrative>Forum for African Women Educationalists</narrative>
+                <narrative xml:lang="fr">Forum des éducatrices africaines</narrative>
             </name>
+            <category>23000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
+        <codelist-item status="withdrawn" activation-date="2010-01-01" withdrawal-date="2010-12-31">
             <code>21011</code>
             <name>
                 <narrative>Global Campaign for Education</narrative>
                 <narrative xml:lang="fr">Campagne mondiale pour l’éducation</narrative>
             </name>
+            <category>21000</category>
         </codelist-item>
-        <codelist-item status="active" activation-date="2015-12-07">
+        <codelist-item status="withdrawn" activation-date="2010-01-01" withdrawal-date="2010-12-31">
```

Comment:
I don’t think this codelist item has been withdrawn. There are two withdrawn entries for this item, but also one active one.

Reply:
Reactivated.